pkg/plugins/manager/loader/loader_test.go

```diff
@@ -378,7 +378,7 @@ func TestLoader_Load(t *testing.T) {
 						Plugins:           []plugins.Dependency{},
 					},
 					Includes: []*plugins.Includes{
-						{Name: "Nginx Memory", Path: "dashboards/memory.json", Type: "dashboard", Role: "Viewer", Slug: "nginx-memory", DefaultNav: true},
+						{Name: "Nginx Memory", Path: "dashboards/memory.json", Type: "dashboard", Role: "Viewer", Slug: "nginx-memory"},
```

Comment:
I updated this purely as this is technically invalid, as both includes have `DefaultNav: true` - we should add validation earlier in the process to formally prevent this (plugin publishing, and also during the loading step)